python/gtsam/tests/test_SfmData.py

```diff
@@ -17,6 +17,7 @@
 import numpy as np
 
 import gtsam
+from gtsam import SfmData, SfmTrack, Point2, Point3
```

Comment:
nit: sorted alphabetically, these should be 
```python
from gtsam import Point2, Point3, SfmData, SfmTrack
```

Reply:
Done